tests/framework/extensions/provisioning/ssh.go

```diff
@@ -0,0 +1,43 @@
+package provisioning
+
+// This file contains all tests that require to ssh into a node to run commands to check things
+// such as any stats, benchmarks, etc. For example, ssh is required to check the cpu usage of a
+// process running on an individual node.
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/rancher/rancher/tests/framework/pkg/nodes"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// For cpuUsageVar, 100 is just a placeholder until we can determine an actual number. Even with cpu usage
+	// spiking it should not go past 100% cpu usage and previous issues concerning this were
+	// hitting around 130% and above
+	cpuUsageVar = 100
+	checkCPU    = "CheckCPU"
+)
+
+// CallSSHTestByName runs through all the ssh tests listed in the cattle-config based on keywords. Check README.md for more info
+func CallSSHTestByName(testName string, nodes *nodes.Node) error {
+	switch testName {
+	case checkCPU:
+		command := "ps -C agent -o %cpu --no-header"
+		output, err := nodes.ExecuteCommand(command)
```

Comment:
should we log the (string) output and maybe nodeIP, just so that we have a record of the test running when looking at jenkins output? Or if that output isn't pretty, maybe just log that we're running this test?

Reply:
agreed

Reply:
I can add it every time just like the podstatus, it's a pretty short message.